curriculum/js0/4.js

```diff
@@ -5,9 +5,9 @@
  */
 
 const solution = (a, b) => {
-  return 0;
+  if (a>b) return a;
+  if (b>a) return b;
```

Comment:
next time, try lookin for ways to simplify your code:

if (a>b) return a;
return b;